client/lib/domains/resolve-domain-status.js

```diff
@@ -44,13 +52,45 @@ export function resolveDomainStatus(
 				};
 			}
 
-			if ( ( ! isJetpackSite || isSiteAutomatedTransfer ) && ! domain.pointsToWpcom ) {
+			if ( hasMappingError ) {
+				const status = translate(
+					"{{strong}}Connection error:{{/strong}} We couldn't verify your connection. Please {{a}}follow this setup again{{/a}}.",
+					{
+						components: {
+							strong: <strong />,
+							a: (
+								<a
+									href={ domainManagementEdit( siteSlug, domain.domain ) }
+									onClick={ ( e ) => e.stopPropagation() }
+								/>
+							),
+						},
+					}
+				);
 				return {
-					statusText: translate( 'Complete setup' ),
-					statusClass: 'status-warning',
+					statusText: translate( 'Connection error' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Publicize Connection Error' )` *ES Score: 7*